pages/k8s/ipv6.md

```diff
@@ -0,0 +1,156 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "IPv6"
+  description: Using IPv6 with Charmed Kubernetes.
+keywords: juju, network, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: ipv6.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+As of Kubernetes 1.19, support for IPv6 is in beta and dual-stack is in alpha.
+Charmed Kubernetes can enable these by including IPv6 CIDRs in the `cidr`
+config for the Calico charm and the `service-cidr` config for the Kubernetes
+Master charm.  These can contain up to two comma-separated values, with the
+first CIDR in the list being the preferred family for pods, or the default for
+services.  You can override the `IPFamily` for a service when creating it.
+
+<div class="p-notification--positive"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Note:</span>
+Calico is the only CNI which supports IPv6 at this time.
+</p></div>
+
+## Example
+
+You can use the following overlay file ([download it here](asset-ipv4-ipv6-overlay])
```

Comment:
I think you mean [download it here][asset-ipv4-ipv6-overlay]?
You need to add the links at the bottom too ;)